ansible/roles/api/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+--- 
+- 
+  name: "Install all required dependencies"
+  yum: 
+    name: 
+      - git
+      - python3
+      - python3-setuptools
+      - python3-pip
+      - python3-virtualenv
+    state: present
+- 
+  name: "Add User"
+  user: 
+    name: robo
+- 
+  git: 
+    dest: /opt/filing13F_searcher
+    repo: "https://github.com/eharley19/filing13F_searcher"
+    version: main
+    force: yes
+  name: "Clone Repo"
+- 
+  file: 
+    owner: robo
+    path: /opt/filing13F_searcher
+    recurse: true
+  name: "Change owner to robo"
+- 
+  become: ansible_common_remote_group
+  become_user: robo
+  name: "Install python setup.py"
+  pip:
+    name: /opt/filing13F_searcher/python
+    virtualenv: ~/venv
+-
+  name: "Upload env template"
```

Comment:
I don't think this file is ever used. You should probably not create it and remove the template.

Reply:
Code removed